packages/cmf/src/sagas/index.md

```diff
@@ -77,6 +77,20 @@ const { data, response } = yield call(http.get, `${API['dataset-sample']}/${data
   + The ```silent``` property to ```true``` avoid that cmf dispatch an action of type ```@@HTTP/ERRORS```.<br/>
   It could be usefull if you want to treat the request error on a specific way only and deal with it within your own saga.
 
+```javascript
+const options = {
+	silent: {
+		notify: false,
+		redirect: false,
+	},
+};
+
+const { data, response } = yield call(http.get, `${API['dataset-sample']}/${datasetId}`, config, options);
+```
+Passing an object instead of a boolean will make cmf dispatch an action of type ```@@HTTP/ERRORS``` and allow you to deal with it in your redirect and notification sagas.<br/>
+If ```notify``` is ```false```, your onHttpErrorNotification saga should send the notification as it is within a ```silent```, if ```true``` it should not.<br/>
+If ```redirect``` is ```false```, your onHttpErrorRedirect saga should redirect to dedicated error page base on the status code as it is within a ```silent```, if ```true``` it should do nothing.<br/>
+
```

Comment:
look like this doc could be outside of setDefault config.
Could you please create a title  for it ?